src/components/build/proposal/questionnaire/brickTitle/brickTitle.tsx

```diff
@@ -93,6 +93,11 @@ const BrickTitle: React.FC<BrickTitleProps> = ({ parentState, canEdit, saveTitle
       <Navigation step={ProposalStep.BrickTitle} onMove={() => saveTitles(parentState)} />
       <Grid container direction="row">
         <Grid item className="left-block">
+
+          {/* <Grid container justify="center">
+            <img alt="titles" src="/images/new-brick/titles.png" className="mobile-view-image" />
+          </Grid> */}
```

Comment:
Commented out code should generally have a date if possible, so that we can delete it if it isn't used for long.